app/app.go

```diff
@@ -596,7 +596,7 @@ func New(
 		ibcTransferModule,
 		gravity.NewAppModule(app.GravityKeeper, app.BankKeeper),
 		leverage.NewAppModule(appCodec, app.LeverageKeeper, app.AccountKeeper, app.BankKeeper),
-		oracle.NewAppModule(appCodec, app.OracleKeeper, app.AccountKeeper, app.BankKeeper, Experimental),
```

Comment:
we can keep the flag, might be needed in the future? ... and we only remove the usage in end blocker

Reply:
imo we should only add it back if necessary, especially since we don't have plans to iterate on the oracle module much more on umee